/**:
  ros__parameters:
    force_x_p: 0.5
    force_x_i: 0.0
    force_x_d: 0.0
    force_y_p: 0.5
    force_y_i: 0.0
    force_y_d: 0.0
    force_z_p: 0.5
    force_z_i: 0.0
    force_z_d: 0.0
    force_yaw_p: 5.0
    force_yaw_i: 0.0
    force_yaw_d: 0.0
